code/modules/client/preferences/entries/character/quirkoptions.dm

```diff
@@ -0,0 +1,186 @@
+/datum/preference/choiced/quirk_prosthetic_limb_location
+	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
+	preference_type = PREFERENCE_CHARACTER
+	db_key = "quirk_prosthetic_limb_location"
+	can_randomize = FALSE
+
+/datum/preference/choiced/quirk_prosthetic_limb_location/init_possible_values()
+	return list(BODY_ZONE_L_ARM, BODY_ZONE_R_ARM, BODY_ZONE_L_LEG, BODY_ZONE_R_LEG)
+
+/datum/preference/choiced/quirk_prosthetic_limb_location/compile_constant_data()
+	var/list/data = ..()
+
+	data[CHOICED_PREFERENCE_DISPLAY_NAMES] = list(
+		BODY_ZONE_L_ARM = "Left Arm",
+		BODY_ZONE_R_ARM = "Right Arm",
+		BODY_ZONE_L_LEG = "Left Leg",
+		BODY_ZONE_R_LEG = "Right Leg",
+	)
+
+	return data
+
+/datum/preference/choiced/quirk_prosthetic_limb_location/apply_to_human(mob/living/carbon/human/target, value)
+	return
+
+/datum/preference/choiced/quirk_prosthetic_limb_location/is_accessible(datum/preferences/preferences, ignore_page = FALSE)
+	if (!..())
+		return FALSE
+	if("Prosthetic Limb" in preferences.all_quirks)
+		return TRUE
+	return FALSE
+
+/datum/preference/choiced/quirk_phobia
+	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
+	preference_type = PREFERENCE_CHARACTER
+	db_key = "quirk_phobia"
+	can_randomize = FALSE
+
+/datum/preference/choiced/quirk_phobia/init_possible_values() // tried to use the SStraumas.phobia_types but it seems that subsystem hasnt initialized before youre able to access character creation
```

Comment:
you can move the init order of SStraumas higher, this should also be a DEFINE or global list otherwise. you should never rely on copypasting something from another source as someone could forget to update it

Reply:
couldn't get moving the init order to work, it's a define now